play/src/pusher/services/MatrixProvider.ts

```diff
@@ -150,6 +142,42 @@ class MatrixProvider {
         }
     }
 
+    async promoteUserToModerator(userID: string, roomID: string): Promise<void> {
+        const actualPowerLevelsResponse = await axios.get(
+            `${MATRIX_API_URI}_matrix/client/r0/rooms/${roomID}/state/m.room.power_levels`,
+            {
+                headers: {
+                    Authorization: "Bearer " + (await this.getAccessToken()),
+                },
```

Comment:
Ideally, we could have a "getAxios()" functino retruning an axios instance configured with the base URI and the authorization headers